src/main/webapp/WEB-INF/static/js/trialmanager/column-filters/column-filters.html

```diff
@@ -40,6 +40,11 @@
 		<input class="form-control" ng-model='filter.value' placeholder="{{filter.placeholder || 'Contains text'}}"/>
 	</study-column-filter>
 </script>
+<script type="text/ng-template" id="studyColumnFilterTextNumberOnly.html">
+	<study-column-filter>
+		<input type="number" maxlength="10" class="form-control" ng-model='filter.value' placeholder="{{filter.placeholder || 'Contains numbers'}}"/>
+	</study-column-filter>
+</script>
```

Comment:
@abatac What do you think if we use `studyColumnFilterList` instead for numbers in _Crosses and selections_ (gid, etc)? for consistency with the inventory tab.

Reply:
I think it makes sense to use studyColumnFilterList for consistency across tables, we will address this during cleanup: IBP-3876